components/script/dom/htmlstyleelement.rs

```diff
@@ -88,6 +90,13 @@ impl HTMLStyleElement {
         let element = self.upcast::<Element>();
         assert!(node.is_connected());
 
+        // Step 4. of <https://html.spec.whatwg.org/multipage/#the-style-element%3Aupdate-a-style-block>
+        let mut type_attribute = self.Type();
+        type_attribute.make_ascii_lowercase();
+        if !type_attribute.is_empty() && type_attribute != "text/css" {
```

Comment:
Spec:
> If element's [type](https://html.spec.whatwg.org/multipage/obsolete.html#attr-style-type) attribute is present and its value is neither the empty string nor an [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive) match for "[text/css](https://html.spec.whatwg.org/multipage/indices.html#text/css)", then return.

Wouldn't this implementation incorrectly not return for `<style type=""`>?

Reply:
Isn't not returning the correct thing to do?

Reply:
Oops, you're right